export/distro/format.go

```diff
@@ -167,3 +168,11 @@ func (af azureFormatter) Format(event *models.Event) []byte {
 	}
 	return msg
 }
+
+type noopFormatter struct {
```

Comment:
I do not see use in declaring an empty struct as a separate type...

Reply:
Slightly disagree here. The type name does impart some additional information i.e the intended usage. In C/C++ quite often see string types, typedef'd to something more meaningful.

Reply:
Especially C++ produces a lot of bloat that go philosophy tries to avoid. Feel free to return empty arrays or strings directly from the code, leave a short comment if necessary, but oplease do not bloat the code.

Every code line is potential bug introduced, keep them at minimum.

Reply:
The empty struct looked strange to me when I first saw the code as a new golang developer. But it's pretty much like a class to implement certain interface, just golang does require you list all the function in the interface in the implementing class.